includes/routes.php

```diff
@@ -107,6 +122,24 @@ function fastwc_api_permission_callback() {
 	return $has_permission;
 }
 
+/**
+ * REST API permissions callback for product attributes
+ *
+ * @return bool
+ */
+function fastwc_api_product_attributes_permission_callback() {
```

Comment:
Question: Would it make sense to generalize the name of this function for reuse in other cases when needed.
Something like fastwc_api_managewc_permission_callback ?

Reply:
Good idea. This is updated.